WooCommerce/WooCommerceUITests/Flows/LoginFlow.swift

```diff
@@ -0,0 +1,15 @@
+import UITestsFoundation
+import XCTest
+
+class LoginFlow {
+
+    // Login with WordPress.com account flow
```

Comment:
Since the function name was made more descriptive, as a step to self-documenting code, I think the comment is not necessary anymore:

```suggestion
```

Reply:
Makes sense, I'll remove this in the next PR.